ledger/sandbox/README.md

```diff
@@ -54,3 +54,35 @@ You can enable debug logging in Sandbox with `sandbox-log-level` system property
 Or when started from Bazel with:
 
     $ bazel run //ledger/sandbox:sandbox-binary -- --log-level=DEBUG $PWD/bazel-bin/ledger/sandbox/Test.dar
+
+# Profiling
```

Comment:
Why is this info in the readme instead of in the docs?

Reply:
Because it is not ready to be used by external people.